docs/src/docs/use-cases.md

```diff
@@ -1,12 +1,40 @@
 
-## How KitOps Gets Used 🛠️
+## Stage 1: Handoff From Development and Production 🤝
 
-Organizations have spent the last 15+ years building up a suite of developer and DevOps tools to make developing software applications and services faster, safer, and easier. As they start working with AI/ML, they are realizing that those tools are not well suited to a world of massive models and even bigger datasets. ModelKits were created by a team who lived that nightmare and wanted others to avoid it.
+Many organizations have AI teams build a [ModelKit](./modelkit/intro.md) for each version of the AI project that is going to staging, user acceptance testing (UAT), or production.
 
-Here’s how we used ModelKits along with the tools we already used in our data science, software development, and infrastructure management teams.
+This ensures that:
+* __Operations teams have all the assets and information they need__ in order to determine how to test, deploy, audit, and manage these new workloads
+* __Organizations are protected against vendor shifts__ in their MLOps and Serving Infrastructure domains (this also gives them negotiating leverage with vendors)
+* __AI versioned packages are held in the same enterprise registry__ as other production assets like containers
+* __Compliance teams have a catalogue of versioned models__ that can be used for EU AI Act or other regulatory reporting
+* __Everyone has a library of immutable and signed ModelKits__ for intellectual property, progress tracking, or other requirements
+
+Using ModelKits as the "gate" between development and production also speeds up the transition between development and staging / production. ModelKit packing can be automated using the Kit [CLI](./cli/cli-reference.md).
+
+At this stage of KitOps usage development artifacts are still solely housed in their various current locations:
+* Datasets in data lakes, databases, files systems, or other similar locations
+* Code in git repositories
+* Models in Jupyter notebooks or MLOps tools
+* Metadata in various locations based on their type
+
+## Stage 2: Storage for all AI Project Versions 💾
+
+Organizations that are more mature in their handling of AI projects, or are subject to extra scrutiny or regulations extend the use of ModelKits to the development phase as well. This solves the currently scattered storage of artifacts.
+
+This ensures that:
+* The organization uses a standards-based storage (OCI 1.1 artifacts) for all their AI/ML project work
+* Data and AI teams who don't work closely together can share artifacts even if they're using different development tools
+* Development artifacts can't be accidentally or maliciously tampered with
+
+This can be a lightweight process that relies on automating the ModelKit packing via the Kit [CLI](./cli/cli-reference.md).
```

Comment:
Consider:  "This can be a lightweight process automated with ModelKit packing via the Kit [CLI]"